### ISSUE TITLE
Cannot compile @nativescript-community/ui-material-core  with latest Android Studio

### ISSUE THREAD

**Issue by cjohn001** [nativescript-community/ui-material-components#350]
Hello @farfromrefug:
I just upgrade to latest Android Studio on my M1 Mac: 

Android Studio Bumblebee | 2021.1.1 Patch 1
Build #AI-211.7628.21.2111.8139111, built on February 1, 2022
Runtime version: 11.0.11+0-b60-7772763 aarch64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
macOS 12.2
GC: G1 Young Generation, G1 Old Generation
Memory: 1280M
Cores: 10
Registry: external.system.auto.import.disabled=true
Non-Bundled Plugins: org.jetbrains.kotlin (211-1.6.10-release-923-AS7442.40)

With the new dev environment I am not able to build the ui-material-core package. I am getting the following error:

```
Downloading https://services.gradle.org/distributions/gradle-7.2.0-bin.zip

Exception in thread "main" java.io.FileNotFoundException: https://downloads.gradle-dn.com/distributions/gradle-7.2.0-bin.zip
        at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1920)
        at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1520)
        at java.base/sun.net.www.protocol.https.HttpsURLConnectionImpl.getInputStream(HttpsURLConnectionImpl.java:250)
        at org.gradle.wrapper.Download.downloadInternal(Download.java:60)
        at org.gradle.wrapper.Download.download(Download.java:45)
        at org.gradle.wrapper.Install$1.call(Install.java:62)
        at org.gradle.wrapper.Install$1.call(Install.java:48)
        at org.gradle.wrapper.ExclusiveFileAccessManager.access(ExclusiveFileAccessManager.java:69)
        at org.gradle.wrapper.Install.createDist(Install.java:48)
        at org.gradle.wrapper.WrapperExecutor.execute(WrapperExecutor.java:107)
        at org.gradle.wrapper.GradleWrapperMain.main(GradleWrapperMain.java:61)
Failed to build plugin @nativescript-community/ui-material-core : 
Error: Command ./gradlew failed with exit code 1
```

### Which platform(s) does your issue occur on?
- Android with Alpha runtime alpha.9
- only emulator tried

### Please, provide the following version numbers that your issue occurs with:

tns --version
8.1.5-next-12-23-2021-1617049163

```
	"dependencies": {
		"@angular/animations": "13.2.1",
		"@angular/common": "13.2.1",
		"@angular/compiler": "13.2.1",
		"@angular/core": "13.2.1",
		"@angular/forms": "13.2.1",
		"@angular/platform-browser": "13.2.1",
		"@angular/platform-browser-dynamic": "13.2.1",
		"@angular/router": "13.2.1",
		"@apollo/client": "3.5.8",
		"@nativescript-community/ui-material-tabs": "6.2.10",
		"@nativescript/angular": "13.0.1",
		"@nativescript/core": "8.1.5",
		"@nativescript/iqkeyboardmanager": "2.0.0",
		"@nativescript/localize": "5.0.4",
		"@nativescript/mlkit-barcode-scanning": "1.0.0-alpha.8",
		"@nativescript/mlkit-core": "1.0.0-alpha.8",
		"@nativescript/secure-storage": "3.0.0",
		"@nativescript/theme": "3.0.2",
		"@nativescript/ui-charts": "file:../mnd-custom-plugins/nativescript-ui-charts/publish/package/nativescript-ui-charts-0.1.10.tgz",
		"apollo-angular": "3.0.0-alpha.2",
		"apollo3-cache-persist": "0.13.0",
		"d3-ease": "3.0.1",
		"graphql": "16.3.0",
		"graphql-tag": "2.12.6",
		"intl": "1.2.5",
		"moment": "2.29.1",
		"nativescript-oauth2-ext": "file:../mnd-custom-plugins/nativescript-oauth2-ext/publish/package/nativescript-oauth2-ext-3.0.0.tgz",
		"nativescript-sqlite": "2.8.6",
		"nativescript-ui-autocomplete": "8.0.1",
		"nativescript-ui-calendar": "file:../mnd-custom-plugins/plugins-ui/dist/packages/ui-calendar/nativescript-ui-calendar-8.0.0.tgz",
		"nativescript-ui-gauge": "8.0.1",
		"reflect-metadata": "0.1.13",
		"rxjs": "7.5.2",
		"uuid": "8.3.2",
		"zone.js": "0.11.4"
	},
	"devDependencies": {
		"@angular-devkit/build-angular": "13.2.2",
		"@angular/compiler-cli": "13.2.1",
		"@graphql-codegen/cli": "2.5.0",
		"@graphql-codegen/fragment-matcher": "3.2.1",
		"@graphql-codegen/introspection": "2.1.1",
		"@graphql-codegen/typescript": "2.4.3",
		"@graphql-codegen/typescript-apollo-angular": "3.3.3",
		"@graphql-codegen/typescript-operations": "2.2.4",
		"@nativescript/android": "alpha",
		"@nativescript/ios": "8.1.0",
		"@nativescript/types": "8.1.1",
		"@nativescript/webpack": "5.0.4",
		"@ngtools/webpack": "13.2.2",
		"@types/d3-ease": "3.0.0",
		"@types/intl": "1.2.0",
		"@types/node": "16.11.6",
		"@types/uuid": "8.3.4",
		"codelyzer": "6.0.2",
		"keycloak-request-token": "0.1.0",
		"rimraf": "3.0.2",
		"sass": "1.49.7",
		"tslint": "^6.1.3",
		"ts-node": "10.4.0",
		"typescript": "4.5.5"
	},
```

**Comment by cjohn001** [comment]
@farfromrefug : Short update on the issue. I just realized that the issue does not arise because of the new Android Studio build tools. Seems like you guys yesterday already released 

"@nativescript/android": "8.2.0-alpha.10"

which my build pulled in the background. The issue with @nativescript-community/ui-material-core building results from the new runtime. If I downgrade to "@nativescript/android": "8.2.0-alpha.9" I can still build the plugin.

**Comment by farfromrefug** [comment]
@cjohn001 not sure what dev env you are talking about. What command and where are you running it?

**Comment by cjohn001** [comment]
@farfromrefug: I observed the issue when I today update my Android Studio to 

Android Studio Bumblebee | 2021.1.1 Patch 1

and therefore initially thought (see initial issue description) the problem with building of the plugin might arise from the build tools of the new Android Studio version I just installed. However, this is not the case. I did an npm run clean with building with the new Android Studio, which at the same pulled down the new "@nativescript/android": "8.2.0-alpha.10" runtime.
As I can say in the meantime, the issue I am seeing is not related to Android Studio, but to the "@nativescript/android": "8.2.0-alpha.10" runtime. If I downgrade to "8.2.0-alpha.9" I can build "@nativescript-community/ui-material-tabs": "6.2.10"
as expected.
So it is unclear to me, if "@nativescript-community/ui-material-tabs" needs to be updated to support "@nativescript/android": "8.2.0-alpha.10" or if it is a bug in the new alpha and not in the ui-material-tabs plugin.

**Comment by farfromrefug** [comment]
@cjohn001 i still dont understand. Where are you using android studio?
N plugin dev does not use android studio so i am at a lost here on where you run gradle.

**Comment by cjohn001** [comment]
@farfromrefug : Ok, for the moment forget about android studio. As I mentioned the issue I am seeing is not related to Android Studio. I try to explain it different:

I try to build my app with

ns debug android --env.aot --no-hmr --emulator

When the app includes the new android alpha runtime 
"@nativescript/android": "8.2.0-alpha.10"
and this plugin 
"@nativescript-community/ui-material-tabs": "6.2.10",

I get the following build error on the console:

```
Downloading https://services.gradle.org/distributions/gradle-7.2.0-bin.zip

Exception in thread "main" java.io.FileNotFoundException: https://downloads.gradle-dn.com/distributions/gradle-7.2.0-bin.zip
        at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1920)
        at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1520)
        at java.base/sun.net.www.protocol.https.HttpsURLConnectionImpl.getInputStream(HttpsURLConnectionImpl.java:250)
        at org.gradle.wrapper.Download.downloadInternal(Download.java:60)
        at org.gradle.wrapper.Download.download(Download.java:45)
        at org.gradle.wrapper.Install$1.call(Install.java:62)
        at org.gradle.wrapper.Install$1.call(Install.java:48)
        at org.gradle.wrapper.ExclusiveFileAccessManager.access(ExclusiveFileAccessManager.java:69)
        at org.gradle.wrapper.Install.createDist(Install.java:48)
        at org.gradle.wrapper.WrapperExecutor.execute(WrapperExecutor.java:107)
        at org.gradle.wrapper.GradleWrapperMain.main(GradleWrapperMain.java:61)
Failed to build plugin @nativescript-community/ui-material-core : 
Error: Command ./gradlew failed with exit code 1
```

As it says:

Failed to build plugin @nativescript-community/ui-material-core : 

I opened this issue. However, when I try to build the same app with
"@nativescript/android": "8.2.0-alpha.9"
the build is successful. 

Therefore, my question was, if the bug I am observing is related to this plugin (@nativescript-community/ui-material-core) which needs to be adopted for the new alpha runtime, or if the bug is related to the new  
"@nativescript/android": "8.2.0-alpha.10" runtime.

In other words, I asked this because I like to know if I opened the issue here in the right place, or if I should rather open it for the android alpha runtime.

I hope my question got more clear now, sorry for the inconvenience.

**Comment by farfromrefug** [comment]
ok! indeed the issue seems to be with the runtime. did not try alpha 10 i think.
is your cli up to date?
for future reference ir is not the plugins which define the gradle version but the cli and the runtime

**Comment by cjohn001** [comment]
@farfromrefug: Ok, thanks for letting me know, I suspected that the plugin might come with some build scripts which could have had a problem with the gradle version. But good to know that the plugins normally should not have issues with gradle. I now added another issue on the android-runtime repo. I leave the issue here open till it is clear what is causing the problem.

Indeed I was not on the latest cli version. However, I just upgraded the cli but the problem stays the same.

**Comment by farfromrefug** [comment]
@cjohn001 on my phone but check.if there is an alpha pf te cli. there.might be one for the alpha runtime. (i use local built of.github master)

**Comment by cjohn001** [comment]
@farfromrefug: Just checked, unfortunately, there is not.

[8.1.5-next-02-03-2022-1791684443] is tagged next and is 2 days old. Unfortunately, there seems to be no 8.2.x alpha.

**Comment by cjohn001** [comment]
@farfromrefug, @triniwiz just provided a runtime update, this fixed the issue. Hence I close the issue here. Thanks for your time!
